src/linter/ui5Types/FixHintsGenerator.ts

```diff
@@ -0,0 +1,816 @@
+	}],
+	// https://github.com/SAP/ui5-linter/issues/561
+	["fesr.setActive", {
+		moduleName: "sap/ui/performance/trace/FESR", exportNameToBeUsed: "setActive",
+	}],
+	["fesr.getActive", {
+		moduleName: "sap/ui/performance/trace/FESR", exportNameToBeUsed: "getActive",
+	}],
+	["fesr.addBusyDuration", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "addBusyDuration",
+	}],
+	["interaction.getActive", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "getActive",
+	}],
+	["interaction.setActive", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "setActive",
+	}],
+	["interaction.notifyStepStart", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "notifyStepStart",
+	}],
+	["interaction.notifyStepEnd", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "notifyStepEnd",
+	}],
+	["interaction.notifyEventStart", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "notifyEventStart",
+	}],
+	["interaction.notifyScrollEvent", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "notifyScrollEvent",
+	}],
+	["interaction.notifyEventEnd", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "notifyEventEnd",
+	}],
+	["interaction.setStepComponent", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "setStepComponent",
+	}],
+	["measure.clearInteractionMeasurements", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "clear",
+	}],
+	["measure.startInteraction", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "start",
+	}],
+	["measure.endInteraction", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "end",
+	}],
+	["measure.filterInteractionMeasurements", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "filter",
+	}],
+	["measure.getAllInteractionMeasurements", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "getAll",
+	}],
+	["measure.getPendingInteractionMeasurement", {
+		moduleName: "sap/ui/performance/trace/Interaction", exportNameToBeUsed: "getPending",
+	}],
+	["fesr.getCurrentTransactionId", {
+		moduleName: "sap/ui/performance/trace/Passport", exportNameToBeUsed: "getCurrentTransactionId",
+	}],
+	["fesr.getRootId", {
+		moduleName: "sap/ui/performance/trace/Passport", exportNameToBeUsed: "getRootId",
+	}],
+	["passport.setActive", {
+		moduleName: "sap/ui/performance/trace/Passport", exportNameToBeUsed: "setActive",
+	}],
+	["passport.traceFlags", {
+		moduleName: "sap/ui/performance/trace/Passport", exportNameToBeUsed: "traceFlags",
+	}],
+	// https://github.com/SAP/ui5-linter/issues/563
+
+	// TODO MB: Possibly missing some replacements, see https://github.com/SAP/ui5-linter/issues/563
```

Comment:
To be discussed, also see https://github.com/SAP/ui5-linter/issues/563#issuecomment-2813018032